src/Mimi.jl

```diff
@@ -14,50 +14,32 @@ using StringBuilders
 
 export
     @defcomp,
-    @modelegate,
     MarginalModel,
     Model,
-    addcomponent,
-    add_connector_comps,
-    add_dimension!,
-    addparameter,
-    compdef,
-    compdefs,
-    compinstance,
-    compkeys,
+    addcomponent,  # push!, add_component!, add_component, addcomp!, add!; could also have multiple functions like insert!, push! etc. instead of before after keywords
+    add_connector_comps, #should this be deleted?  Docs indicate we need it ...
```

Comment:
This gets automatically called within each `@defcomp`, but I think we still need it at the end of the API for building models.

Reply:
We could make `add_connector_comps` the first step of `build(m::Model)`. This would allow us to remove it from `@defcomp` and alleviate the user from calling it interactively. Any objections?

Reply:
I like that!  I'm not fully comfortable with the internals of editing a built model yet ... but I assume any time a big change is made which requires a new call to `add_connector_comps`, we probably `decache` and then `build` again.  That question doesn't affect this decision, just trying to be clear when I'm editing docs.